src/main/java/com/ericsson/ei/handlers/MongoDBHandler.java

```diff
@@ -318,7 +318,6 @@ private void closeMongoDbConecction() {
         if (mongoClient != null) {
             mongoClient.close();
         }
-        mongoClient = null;
```

Comment:
I don't like this solution especially when there are no tests to prove it. But otherwise it feels wrong

Reply:
It is closed in the if statement above this. Is that not enough?

Reply:
Yes, but as we discussed, this MongoDb will be removed/rewritten in Anders pull request in future release.

Reply:
I don't like to change code without tests. So I am still against it

Reply:
I have tested it locally, by taking down and up the MongoDb instance manaully and MongoDb connection comes up.
Tested both before this change and after the change.
Since this code will be removed, I will not spend time on writing test for it.
Then someone else can write test or we skip this change and close the PR.

Reply:
I will not fix this issue in the PR that is up now, it is already large, but I may take the bug story and implement a more solid solution.

Reply:
I would vote to close the PR then